Classes/Domain/Model/Changes/Remove.php

```diff
@@ -104,4 +104,17 @@ public function apply(): void
             $this->feedbackCollection->add($updateParentNodeInfo);
         }
     }
+
+    private function getRemovalAttachmentPoint(): ?NodeAggregateId
+    {
+        $subgraph = $this->contentRepositoryRegistry->subgraphForNode($this->subject);
+
+        if ($this->subject->nodeType->isOfType(NodeTypeNameFactory::NAME_DOCUMENT)) {
```

Comment:
```suggestion
        if ($this->subject?->nodeType->isOfType(NodeTypeNameFactory::NAME_DOCUMENT)) {
```

otherwise one wouldnt be able to remove nodes without nodeType

Reply:
Is `subject` nullable here or is it rather `subject->nodeType`? So:

```php
if ($this->subject->nodeType?->isOfType(NodeTypeNameFactory::NAME_DOCUMENT)) {
```

Reply:
no sorry of course the latter :)

Reply:
Adjusted :)